### PR TITLE
Updated to 2.4.3

### PR DESCRIPTION
**Pull request by niemasd**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### Updates to existing formula: have you

- [x] Not altered the `bottle` section?
- [x] Checked that the tests still pass?

### CODE REVIEW

File: varscan.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ class Varscan < Formula
   depends_on :java
 
   def install
-    jar = "VarScan.v2.3.7.jar"
+    jar = "VarScan.v2.4.3.jar"
```
Comment (jonchang) [inline varscan.rb]: Try `"VarScan.v#{version}.jar"` so we don't have to change this every time it updates

File: varscan.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Varscan < Formula
   desc "Variant detection in massively parallel sequencing data"
-  homepage "https://varscan.sourceforge.io/"
-  url "https://downloads.sourceforge.net/project/varscan/VarScan.v2.3.7.jar"
-  sha256 "1060f823fc82a182fcc4ae5824a3639b35ff6d969657c536f709f24b2324ed12"
+  homepage "http://dkoboldt.github.io/varscan/"
```
Comment (jonchang) [inline varscan.rb]: Please fix this audit and then run `brew audit varscan --strict --online` and fix all other errors that pop up:

```
homebrew/science/varscan:
  * Please use https:// for http://dkoboldt.github.io/varscan/
```

Comment (niemasd) [inline varscan.rb]: Done!

File: varscan.rb (hunk 1 above)
Comment (niemasd) [inline varscan.rb]: Done, thanks for the suggestion! I wasn't aware of this.

**Comment by niemasd** [comment]
I've addressed the review comments. Sorry about that, thanks for the suggestions! I'm a bit new to this, so I appreciate the tips!

**Comment by jonchang** [comment]
Thanks @niemasd, this has been merged.

**Pull request closed without merging**